queries.sql

```diff
@@ -14,3 +14,61 @@ SELECT * FROM animals WHERE name <> 'Gabumon';
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.40 AND 17.30;
 
+-- Update and Delete changes to species
+BEGIN;
+
+UPDATE animals SET species = 'unspecified';
+
+SELECT * FROM animals;
+
+ROLLBACK;
```

Comment:
- [ ] After rollback, please verify that the changes were undone e.g 
```
ROLLBACK;

   SELECT species from animals; -- verify that change was undone
```

Reply:
Thank you. Changes made.